src/i_winmusic.c

```diff
@@ -49,7 +49,7 @@ enum
     RESET_TYPE_XG,
 };
 
-char *winmm_midi_device = NULL;
+char *winmm_midi_device = "";
```

Comment:
This causes a constness issue.

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1642/commits/7929bbc56c036a0261b5d28bdf4782d0e7bb6098